spec/system/interface_color_selector_spec.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+describe "Interface color selector", type: :system do
+  let!(:light_scheme) { ColorScheme.find_by(base_scheme_id: "Solarized Light") }
+  let!(:dark_scheme) { ColorScheme.find_by(base_scheme_id: "Dark") }
+
+  let(:selector_in_sidebar) do
+    PageObjects::Components::InterfaceColorSelector.new(find(".sidebar-footer-actions"))
```

Comment:
We should always do these `find` calls on demand otherwise you can end up with stale elements, just pass in the CSS class